src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (a) => {
+    return (b) => {
+      if (counter >= 3 && counter % 2 !== 0) {
+        counter++;
+
+        return 'Bzzz... Error!';
+      };
+
+      counter++;
```

Comment:
Pieces of repeatable logic can be moved to a place where they will be called regardless of conditions

```suggestion
    return (b) => {
      counter++;
      
      if (counter >= 3 && counter % 2 !== 0) {
        return 'Bzzz... Error!';
      };
```